src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/CaseNotesApiService.kt

```diff
@@ -69,19 +78,20 @@ class CaseNotesApiService(
     }
   }
 
-  private fun buildMessage(
+  fun buildMessage(
     person: PersonEntity?,
     referral: ReferralEntity,
     referralStatusUpdate: ReferralStatusUpdate,
+    message: String,
   ): String {
-    log.warn("Request received for creating case notes :${referral.id} $referralStatusUpdate")
+    log.info("Request received for creating case notes :${referral.id} $referralStatusUpdate")
     val course = referral.offering.course
     val orgName = organisationRepository.findOrganisationEntityByCode(referral.offering.organisationId)?.name
+    val progDescMessage = "Referral to ${course.name}: ${course.audience} strand at $orgName \n"
```

Comment:
Could you please expand this variable name to describe what it represents? 

E.g programmeDescriptionMessage or similar?